exercises/nth-prime/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub fn nth(n: u32) -> Result<u32, ()> {
```

Comment:
`rustc` should complain when asked to compile this stub, because `n` isn't actually used in the function. (`rustc` doesn't know that this is a stub function designed to be replaced by students!) 

While we don't have tooling in place which requires that the stubs and examples compile without warnings, the goal is to get there eventually. As such, please change `n` to either `_` or `_n`.